src/python/pants/backend/native/subsystems/native_toolchain.py

```diff
@@ -83,7 +84,20 @@ def _get_platform_specific_subsystems(cls):
   def subsystem_dependencies(cls):
     prev = super(NativeToolchain, cls).subsystem_dependencies()
     cur_platform_subsystems = cls._get_platform_specific_subsystems()
-    return prev + tuple(sub.scoped(cls) for sub in cur_platform_subsystems)
+    return prev + tuple(sub.scoped(cls) for sub in cur_platform_subsystems) + (
+      GLibc.scoped(cls),
+    )
+
+  @memoized_method
+  def glibc_dir(self):
+    # FIXME: this can be removed once #5815 lands (GCC should be made to depend on GLibc on Linux).
```

Comment:
It's not clear what the "this" is that can be removed.

Reply:
Made it a lot more specific in c3aa525, thanks!